pkg/api/api.pb.go

```diff
@@ -171,6 +171,61 @@ func (ContainerState) EnumDescriptor() ([]byte, []int) {
 	return file_pkg_api_api_proto_rawDescGZIP(), []int{1}
 }
 
+type LogRequest_Level int32
+
+const (
+	LogRequest_LEVEL_UNSPECIFIED LogRequest_Level = 0
+	LogRequest_LEVEL_DEBUG       LogRequest_Level = 1
+	LogRequest_LEVEL_INFO        LogRequest_Level = 2
+	LogRequest_LEVEL_WARN        LogRequest_Level = 3
+	LogRequest_LEVEL_ERROR       LogRequest_Level = 4
+)
+
+// Enum value maps for LogRequest_Level.
+var (
+	LogRequest_Level_name = map[int32]string{
+		0: "LEVEL_UNSPECIFIED",
+		1: "LEVEL_DEBUG",
+		2: "LEVEL_INFO",
+		3: "LEVEL_WARN",
+		4: "LEVEL_ERROR",
+	}
+	LogRequest_Level_value = map[string]int32{
+		"LEVEL_UNSPECIFIED": 0,
+		"LEVEL_DEBUG":       1,
+		"LEVEL_INFO":        2,
+		"LEVEL_WARN":        3,
+		"LEVEL_ERROR":       4,
+	}
+)
+
+func (x LogRequest_Level) Enum() *LogRequest_Level {
+	p := new(LogRequest_Level)
+	*p = x
+	return p
+}
+
+func (x LogRequest_Level) String() string {
+	return protoimpl.X.EnumStringOf(x.Descriptor(), protoreflect.EnumNumber(x))
+}
+
+func (LogRequest_Level) Descriptor() protoreflect.EnumDescriptor {
+	return file_pkg_api_api_proto_enumTypes[2].Descriptor()
+}
+
+func (LogRequest_Level) Type() protoreflect.EnumType {
+	return &file_pkg_api_api_proto_enumTypes[2]
+}
+
+func (x LogRequest_Level) Number() protoreflect.EnumNumber {
+	return protoreflect.EnumNumber(x)
+}
+
+// Deprecated: Use LogRequest_Level.Descriptor instead.
```

Comment:
needed?

Reply:
That's generated code which we don'lt have influence on.